tfe/data_source_workspace.go

```diff
@@ -0,0 +1,103 @@
+package tfe
+
+import (
+	"fmt"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceTFEWorkspace() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFEWorkspaceRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"organization": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"auto_apply": &schema.Schema{
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"ssh_key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"queue_all_runs": &schema.Schema{
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"terraform_version": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"working_directory": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"vcs_repo": &schema.Schema{
+				Type:     schema.TypeSet,
```

Comment:
Will update it to a list 👍

Not sure if people need it, but since the info is available I think we should just return it. Pretty sure that if I remove it that within a few days or weeks someone comes to ask for it 😉